### PR TITLE
fix(application-system-form): Checkbox ordering and look consolidation

### PR DESCRIPTION
**Pull request by HjorturJ**
<a href="https://app.asana.com/0/1207402031871459/1208714552607675/f" target="_blank">Task Link</a>
## What

Making checkbox fields align the same way other components do when there are multiple of them in a row and adjusting the checkbox height to match other similar fields like the radio buttons

## Why

Because it looks right.

## Screenshots / Gifs

Before 
image

After
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout handling for checkbox and radio form fields based on their width and type.
  
- **Bug Fixes**
	- Removed unnecessary debug logging from the RadioFormField component.

- **Style**
	- Adjusted padding for the Checkbox component to improve visual consistency.

### CODE REVIEW

**Review by norda-gunni — APPROVED** [review]
LGTM

File: libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts
Comment (Toti91) [inline libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts:61]: This change LGTM but normally when changing design elements in the ui-core lib we should get a designer to confirm the changes (Össi)

**Review by Toti91 — COMMENTED** [review]
One comment, also this seems to be [reverting changes](https://github.com/island-is/island.is/pull/16467) done by @jonnigs a month ago we should get him to confirm

File: libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts
Comment (HjorturJ) [reply libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts:61]: Ah ofc, sounds good, will do 😄

Comment (HjorturJ) [reply libs/island-ui/core/src/lib/Checkbox/Checkbox.css.ts:61]: Össi has given his approval ❤️

**Review by Toti91 — APPROVED** [review]

**Pull request merged**
